spec/spec_helper_acceptance.rb

```diff
@@ -5,7 +5,18 @@
   when 'Debian'
     install_module_from_forge_on(host, 'puppetlabs-apt', '>= 4.1.0 < 9.0.0')
   when 'RedHat'
-    install_module_from_forge_on(host, 'garethr-erlang', '>= 0.3.0 < 1.0.0')
+    if fact_on(host, 'os.release.major').to_i < 8
+      # In Cent OS <8, rabbitmq-server is available in EPEL.
+      install_module_from_forge_on(host, 'puppet/epel', '>= 3.0.0')
+    else
+      # In Cent OS >=8, rabbitmq-server is no longer available in EPEL
+      on host, 'curl -s https://packagecloud.io/install/repositories/rabbitmq/erlang/script.rpm.sh | sudo bash'
```

Comment:
that's quite hacky. can't we implement it into the puppet module to configure the repository?

Reply:
This is now taken care by the puppet module.